packages/lambda-powertools-pattern-basic/index.js

```diff
@@ -9,7 +9,7 @@ const FUNCTION_NAME = process.env.AWS_LAMBDA_FUNCTION_NAME
 const FUNCTION_VERSION = process.env.AWS_LAMBDA_FUNCTION_VERSION
 const ENV = process.env.ENVIRONMENT || process.env.STAGE
 
-if (!process.env.DATADOG_PREFIX) {
+if (typeof process.env.DATADOG_PREFIX === 'undefined') {
```

Comment:
do we need `typeof` here? would `process.env.DATADOG_PREFIX === undefined` suffice?

Reply:
Personally would prefer `typeof` to avoid comparison with different types but in this case it's enough `===`.